tests/GeniusYield/Test/Providers/Mashup.hs

```diff
@@ -32,12 +32,12 @@ providersMashupTests configs =
            delayBySecond
            systemStart <- gyGetSystemStart provider
            delayBySecond
-           Api.EraHistory mode interpreter <- gyGetEraHistory provider  -- `mode` here doesn't appear to have `Eq` instance, comparing via it's `Show` instance should be fine.
+           Api.EraHistory interpreter <- gyGetEraHistory provider  -- `mode` here doesn't appear to have `Eq` instance, comparing via it's `Show` instance should be fine.
```

Comment:
We'll have to delete this comment at a later point as now it would be confusing since there is no `mode`.

Reply:
The comment seems to have been outdated for awhile. I removed it.